src/ApiPort/ApiPort/ApiPort.props

```diff
@@ -3,20 +3,29 @@
     <OutputType>Exe</OutputType>
     <AssemblyName>ApiPort</AssemblyName>
     <RootNamespace>ApiPort</RootNamespace>
-    <!--
-      netcoreapp target must be first to avoid a ResXFileCodeGenerator issue
-      (tracked at https://github.com/dotnet/project-system/issues/1519)
-    -->
-    <TargetFrameworks>netcoreapp2.1;net461</TargetFrameworks>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
     <GenerateBindingRedirectsOutputType>true</GenerateBindingRedirectsOutputType>
```

Comment:
Verify proxy support now that you're removing this. I believe this is the only thing that would not be supported (@conniey can you think of anything else)?

Reply:
Added back desktop support to make sure not regress proxy feature.